### PR TITLE
ci: mynewt: Use newt for downloading dependencies

### PR DESCRIPTION
**Pull request by sjanc**
This makes sure that Mynewt targets have all required dependencies downloaded.

### CODE REVIEW

File: ci/mynewt_install.sh [hunk 1]
```diff
@@ -30,13 +30,17 @@ install_newt() {
...
 
-    # nrfx is now taken from original repository
-    git clone --depth=1 --branch v2.8.0 https://github.com/NordicSemiconductor/nrfx.git repos/nordic-nrfx
+    pushd /tmp/mynewt
+    newt upgrade --shallow=1
+    [[ $? -ne 0 ]] && exit 1
+
+    popd
+    rm /tmp/mynewt/repos/mcuboot -rf
+    cp -r /tmp/mynewt/repos .
```
Comment (utzig) [inline ci/mynewt_install.sh:43]: What's the problem of using `mv`?

Comment (sjanc) [reply ci/mynewt_install.sh:43]: no problem, I'll update it to use mv

**Review by d3zd3z — CHANGES_REQUESTED** [review]

**Comment by nordicjm** [comment]
Rebase needed

**Comment by utzig** [comment]
I tried updating this PR with the suggested changes but it is broken because `newt new` creates a project that requires Mynewt 1.11.0 release which is not available yet. The new release is being made these week afaik. I suggest using #1777 right now and come back to this PR after Mynewt is fully released.

**Comment by sjanc** [comment]
This should work now (you were unlucky to hit in the middle of me tagging repos for release;-)

**Comment by sjanc** [comment]
Hi,

Why #1777 was merged instead of this?

**Pull request closed without merging**